frontend/public/components/monitoring/alerting.tsx

```diff
@@ -263,13 +264,75 @@ export const StateCounts: React.FC<{ alerts: PrometheusAlert[] }> = ({ alerts })
     <>
       {states.map((s) => (
         <div className="monitoring-icon-wrap" key={s}>
-          {stateIcons[s]} {counts[s]} {_.startCase(s)}
+          <AlertStateIcon state={s} /> {counts[s]} {_.startCase(s)}
         </div>
       ))}
     </>
   );
 };
 
+const PopoverField: React.FC<{ body: React.ReactNode; label: string }> = ({ body, label }) => (
+  <Popover headerContent={label} bodyContent={body}>
+    <Button variant="plain" className="details-item__popover-button">
+      {label}
+    </Button>
+  </Popover>
+);
+
+const alertStateHelp = (
+  <>
+    <p>
```

Comment:
Using a `dl` would be better for accessibility, but I have a feeling it will be annoying to style like we want. (I'm OK leaving it as-is if `dl` causes problems.)

Reply:
Addressed by https://github.com/openshift/console/pull/6170